analyzers/SonarAnalyzer.sln

```diff
@@ -6,7 +6,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Common Solution Items", "Co
 	ProjectSection(SolutionItems) = preProject
 		..\.editorconfig = ..\.editorconfig
 		src\AssemblyInfo.Shared.cs = src\AssemblyInfo.Shared.cs
-		..\azure-pipelines-external.yml = ..\azure-pipelines-external.yml
```

Comment:
Bye

Reply:
We can delete this too, right? To clean up. 
https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build?definitionId=99